tests/test_pois.py

```diff
@@ -185,7 +208,18 @@ def test2_4_invalid_story_id(self):
         response = r.json()
         self.assertEqual(response['code'], 404)
 
-    def test2_5_invalid(self):
+    def test2_5_invalid_delete(self):
         r = requests.delete('http://127.0.0.1:5000/pois/{}'.format(poi_id_invalid))
         response = r.json()
         self.assertEqual(response['code'], 404)
+
+    def test3_1_invalid_create_none(self):
+        r = requests.post('http://127.0.0.1:5000/pois', json=poi_create_invalid_none)
+        response = r.json()
+        self.assertEqual(response['code'], 422)
+
+    def test3_2_invalid_update_none(self):
```

Comment:
Disregard final test (3_2). It will have to be moved up to between 1_5 and 1_6 to be a meaningful test.

Reply:
This is actually valid. It actually should have status code 200 (name will be changed to test3_2_update_none, and the JSON will be poi_update_none)